src/Robot.cpp

```diff
@@ -9,6 +9,7 @@
 #include "Subsystems/Lifter.h"
 
 #include "Commands/SetShooter.h"
+#include "Commands/TankDrive.h"
```

Comment:
The use of TankDrive has been removed from Robot.cpp. I don't think there are any changes needed here.